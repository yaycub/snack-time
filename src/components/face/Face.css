.Face {
  padding: 2rem;
  text-align: center;
  font-size: 70vh;
}
