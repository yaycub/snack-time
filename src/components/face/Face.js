import React from 'react';
import PropTypes from 'prop-types';
import styles from './Face.css';

const Face = ({ emoji }) => <p className={styles.Face}>{emoji}</p>;

Face.propTypes = {
  emoji: PropTypes.string.isRequired
};

export default Face;
