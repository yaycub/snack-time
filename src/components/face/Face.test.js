import React from 'react';
import { shallow } from 'enzyme';
import Face from './Face';

describe('Face component', () => {
  it('renders Face', () => {
    const wrapper = shallow(<Face emoji="ðŸ˜€" />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
