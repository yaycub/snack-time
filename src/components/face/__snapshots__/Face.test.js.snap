// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Face component renders Face 1`] = `
<p
  className="Face"
>
  ðŸ˜€
</p>
`;
